client/src/components/Heroes/HeroesCountBadge.tsx

```diff
@@ -0,0 +1,17 @@
+import { Badge, Tooltip, Avatar } from 'antd';
+import { HeroesRadarBadgeDto } from 'api';
+import heroesBadges from 'configs/heroes-badges';
+
+function HeroesCountBadge({ badge: { badgeId, badgeCount } }: { badge: HeroesRadarBadgeDto }) {
+  return (
+    <div style={{ margin: 5, display: 'inline-block' }}>
+      <Badge count={badgeCount}>
+        <Tooltip title={(heroesBadges as any)[badgeId].name}>
```

Comment:
please try to type properly, we avoid using `any`

Reply:
> please try to type properly, we avoid using `any`

Oh thank you for catching this one, I copypasted from the previous component and didn't see that

Reply:
Fixed. Add typing to heroesBadges, remove type assertion in all files with such piece of code